packages/address/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@shopify/address",
-  "version": "2.7.17",
+  "version": "2.8.17",
```

Comment:
Again, same as https://github.com/Shopify/quilt/pull/1302/files#r388275151

Reply:
Also, this version would be `2.8.0`

Reply:
Right good catch! Totally slipped my mind.